src/GenAPI/Microsoft.DotNet.GenAPI/CSharpSyntaxWriter.cs

```diff
@@ -33,21 +35,21 @@ public CSharpSyntaxWriter(string? exceptionMessage)
         public override SyntaxNode? VisitInterfaceDeclaration(InterfaceDeclarationSyntax node)
         {
             node = (InterfaceDeclarationSyntax)base.VisitInterfaceDeclaration(node)!;
```

Comment:
Using `?` instead of `!` here might be preferable, since it will early return when `VisitInterfaceDeclaration` returns a null syntax node.

Same for below.